### PR TITLE
Adds GitHub default tracking section

### LINKED ISSUE #12: Default tracking section
Add a default tracking section to the Github page.

[tracking](https://gist.github.com/canton7/1053846)

### PR DESCRIPTION
**Pull request by SebastianNeiswanger**
Closes #12. Lays out how to set up default tracking in a new section. Tested by yours truly to know it works.

### CODE REVIEW

File: Dev-Tools/github/tool.md [hunk 1]
```diff
@@ -58,6 +58,15 @@ Are you sure you want to continue connecting (yes/no/[fingerprint])?
 12. Enter `yes`.
 13. If you get something like `Hi username! You've successfully authenticated, but GitHub does not provide shell access.` the setup was successful.
 
+### Default Tracking (optional)
+
+Default tracking enables your local installation of git to always point new branches to the origin. It removes the need to explicitly tell the branch.
+
+1. Open a terminal
+2. Check your git version with `git --version`. You need to have version 2.37+
+   1. If you are not updated make sure to update git before continuing and refresh your terminal
```
Comment (ashlyndevries) [inline Dev-Tools/github/tool.md]: Include steps on how to update git.

**Review by ashlyndevries — CHANGES_REQUESTED** [review]

File: Dev-Tools/github/tool.md (hunk 1 above)
Comment (SebastianNeiswanger) [reply Dev-Tools/github/tool.md]: I have added a section for that now

**Review by ashlyndevries — CHANGES_REQUESTED** [review]
@SpBills, LGTY?

**Review by ashlyndevries — APPROVED** [review]
LGTM

**Pull request merged**